src/unix/process.c

```diff
@@ -383,6 +391,74 @@ static void uv__process_child_init(const uv_process_options_t* options,
 #endif
 
 
+int uv_get_children_pid(pid_t ppid, uint32_t** proc_list, int* proc_count) {
+  uint32_t* temp = uv__malloc(0);
```

Comment:
You can do `temp = NULL` instead of `uv__malloc(0)`, `realloc` should behave like `malloc` when its first parameter is `NULL`. I think otherwise you’d have to `free(temp)` on all returns?